### PR TITLE
Fix Query Serving Docs

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR checks the validity of the current query serving docs

### Related Issues
- https://github.com/vitessio/website/issues/1240

### CODE REVIEW

File: content/en/docs/16.0/reference/features/vindexes.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Vindex Type | Cost
 
 In the case of a simple select, Vitess scans the WHERE clause to match references to Vindex columns and chooses the best one to use. If there is no match and the query is simple without complex constructs like aggregates, etc., it is sent to all shards.
 
-Vitess can handle more complex queries. For now, refer to the [design doc](https://github.com/vitessio/vitess/blob/main/doc/V3HighLevelDesign.md) for background information on how it handles them.
+Vitess can handle more complex queries with the new Gen4 planner.
```
Comment (GuptaManan100) [inline content/en/docs/16.0/reference/features/vindexes.md:173]: In this vindexes page, there is a list of supported vindexes but that doesn't list the newly introduced `multicol` vindex.

Comment (GuptaManan100) [inline content/en/docs/16.0/reference/features/vindexes.md:173]: I couldn't find a doc to link for the gen4 planner. The previously linked design doc is for v3, which I don't think is valid anymore.

**Comment by GuptaManan100** [comment]
This PR also has the fix to the presentations page to fix the width and height of the pdfs there - 

Before -
Screenshot 2022-12-16 at 3 01 27 PM

After - 
Screenshot 2022-12-20 at 10 50 21 AM

cc - @rsajwani

File: content/en/docs/16.0/reference/features/vindexes.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/16.0/reference/features/vindexes.md:173]: An entry for it is present.

File: content/en/docs/15.0/reference/features/connection-pools.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ Flag: `--transaction_limit_per_user`
 ### vtgate system settings
 Flag: `--enable_system_settings`
 
-This vtgate flag allows clients to modify a [subset of system settings](https://github.com/vitessio/vitess/blob/main/go/vt/sysvars/sysvars.go#L166-L211) on the MySQL. This is done using the mechanism of [reserved connection](../../query-serving/reserved-conn/#enabling-reserved-connections).
+This vtgate flag allows clients to modify a [subset of system settings](https://github.com/vitessio/vitess/blob/main/go/vt/sysvars/sysvars.go#L174-L217) on the MySQL. This is done using the mechanism of [reserved connection](../../query-serving/reserved-conn/#enabling-reserved-connections).
```
Comment (frouioui) [inline content/en/docs/15.0/reference/features/connection-pools.md:125]: I feel like this is not very maintainable in the long term. We will have to remember to change this specific line in the documentation each time we change the `sysvars.go` file.
Perhaps we should link the user to the file and say to look under the `UseReservedConn` slice.

Comment (GuptaManan100) [reply content/en/docs/15.0/reference/features/connection-pools.md:125]: Yep, this too should be generated. I'll add an item for release-17.0 but for now I think this is fine.

Comment (GuptaManan100) [reply content/en/docs/15.0/reference/features/connection-pools.md:125]: https://github.com/vitessio/website/issues/1302#issuecomment-1369410272

Comment (frouioui) [reply content/en/docs/15.0/reference/features/connection-pools.md:125]: Sounds good to me!

File: content/en/docs/15.0/reference/features/mysql-replication.md [hunk 3]
```diff
@@ -12,23 +12,25 @@ Vitess makes use of MySQL Replication for both high availability and to receive
...
 
 Thus, you can survive sudden primary failure without losing any transactions that were reported to clients as completed. In MySQL 5.7+, this guarantee is strengthened slightly to preventing loss of any transactions that were ever **committed** on the original primary, eliminating so-called [phantom reads](http://bugs.mysql.com/bug.php?id=62174).
 
 On the other hand these behaviors also give a requirement that each shard must have at least 2 tablets with type *replica* (with addition of the primary that can be demoted to type *replica* this gives a minimum of 3 tablets with initial type *replica*). This will allow for the primary to have a semi-sync acker when one of the replica tablets is down for any reason (for a version update, machine reboot, schema swap or anything else).
+These requirements will changed based on the durability policy.
```
Comment (frouioui) [inline content/en/docs/15.0/reference/features/mysql-replication.md:27]: `will changed` -> `will change`

**Review by frouioui — APPROVED** [review]

**Review by rsajwani — APPROVED** [review]
LGTM

**Pull request merged**